### ISSUE TITLE
Question: Datepicker?

### ISSUE THREAD

**Issue by paulohsa32** [justindujardin/ng2-material#266]
Hi, 
i know the project is relactively new but are in your plans add a mdDatepicker component?

**Comment by Gregcop1** [comment]
Yes it could be a nice feature but a tough one. There's a lot of work to do that and we don't really have a planning on next components. Do you want to give us some help ?

**Comment by leocaseiro** [comment]
I have some examples from angular 1 that might help to start:

https://github.com/wdullaer/MaterialDateTimePicker

https://github.com/alenaksu/mdPickers

https://github.com/princemaple/md-datetime

https://github.com/puranjayjain/md-date-time-picker
